.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+# Description of PR purpose/changes
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Optional Fixes # (issue)
+
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Instructions for how to test the PR
+
+# Dev Checklist:
+
```

Comment:
For the dev checklist, do we want to include all of this or just what we agreed on as a team? 
https://sites.google.com/d/1hHn4AjWF19JdEjxkyMmrkBYIcMFZWxpp/p/1_xSUWHepJ3fRf64k4VC-Ex-2eKNPTebe/edit?pli=1

Reply:
I added that as a link